extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageUtil.java

```diff
@@ -452,6 +452,8 @@ private static ConvertedArrowSchema convertArrowSchema(
                 // TODO (deephaven-core#3403) widen this check for better assurances
```

Comment:
We need to change the `boolean.class` on line 450, too.

Reply:
This request is wrong; I now think we should check for both `boolean.class` and `Boolean.class`. This is validating that booleans are being sent using arrow's proper wire format. In the code I can see that both, as deephaven column types, use the same format, so we might as well.